drivers/SmartThings/zigbee-smoke-detector/capabilities/selfCheck.yaml

```diff
@@ -0,0 +1,30 @@
+id: stse.selfCheck
+version: 1
+status: proposed
+name: Self Check
+ephemeral: false
+attributes:
+  selfCheckState:
+    schema:
+      type: object
+      properties:
+        value:
+          type: string
+          enum:
+            - idle
+            - selfChecking
+            - selfCheckCompleted
+      additionalProperties: false
+      required:
+        - value
+    setter: startSelfCheck
+commands:
+  startSelfCheck:
+    name: startSelfCheck
+    arguments:
+      - name: state
+        optional: false
+        schema:
+          type: string
+          enum:
+            - startSelfCheck
```

Comment:
Hi, @greens 

> The state should be reported by the attribute. This is the command.

Yes


> The current driver doesn't even support the ability to set the state to "idle" or "selfCheckCompleted", which is why I think this command accepting these values is not necessary.

Ahh. I got what you mean. Yes, a single "startSelfCheck" command is enough.
This is the latest App ScreenShot.
![Screenshot_20240201_180256_SmartThings](https://github.com/SmartThingsCommunity/SmartThingsEdgeDrivers/assets/40796353/491a218f-314e-487f-9334-1a2547c2358f)


I have tried defining the command like this:
```
commands:
  startSelfCheck:
    name: startSelfCheck
    arguments: []
```
But this definition broke my presentation.yaml...
I use "list" as the displayType in my presentation.yaml because I hope the device card 
could display status and provide an option to issue command.


"list" displayType must include "alternatives". If I use this style definition, I don't know how to fill "alternatives".

Reply:
> I'm not very familiar with it, but I would expect that if you change the command to no longer take arguments then you will need to update the presentation.

Due to Seongyeol's help, I solved the problem I mentioned above. 

Hi @Hongming6 
Please change selfCheck yaml to this. (also in PR#1160)
```
id: stse.selfCheck
version: 1
status: proposed
name: Self Check
ephemeral: false
attributes:
  selfCheckState:
    schema:
      type: object
      properties:
        value:
          type: string
          enum:
            - idle
            - selfChecking
            - selfCheckCompleted
      additionalProperties: false
      required:
        - value
commands:
  startSelfCheck:
    name: startSelfCheck
    arguments: []
```

Reply:
Get it,modified.